Source/GSXib5KeyedUnarchiver.m

```diff
@@ -287,6 +288,18 @@ + (void) initialize
                                            @"string", @"NS.relative",
                                            @"canPropagateSelectedChildViewControllerTitle",
                                                    @"NSTabViewControllerCanPropagateSelectedChildViewControllerTitle",
+                                           @"rowAlignment", @"NSGrid_alignment",
```

Comment:
We should rather add a mechanism to do this mapping automatically. similar to the one I already included to add the class name without underscore.

Reply:
Good suggestion.  I was thinking about that, but since this is the first class I have seen that does it I wasn't sure if I should add such a thing.   I will make the changes.

Reply:
Where is this happening in the Xib code?

Reply:
I would suggest to put that code into the method alternateName:, we could for example scan for "_" and remove everything before that. You just need to make sure this won't interfere with other decoding mechanisms. What ever you do, please document it on the method. The behaviour is already complicated enough.

Reply:
okay, I will do this.   I am going to get it a little closer to working and then optimize in the way you suggest.

Reply:
Are you going to work on this or leave it that way?

Reply:
I am not sure if it's really a clean mapping.   It's not just a matter of putting the class name in front like so @"%@_%@", className, attributeName...   the attribute in the XIB is alignment, but the one in the nib is rowAlignment.   Granted there are not that many exceptions, but I was going to ask you if this seemed worth it.  If you think it is, I will do it.

Reply:
OK, so we leave it for now.

Reply:
OK